### DISCUSSION TITLE
Has someone experienced v2rayNG on Nvidia Shield?

### DISCUSSION THREAD

**Discussion by asaarashi** [2dust/v2rayNG#1423] [Q&A] [answered]
I intend to buy an Nvidia shield but currently, I am living in China and must bypass the wall to make it work. I searched it from the threads and found a related discussion https://github.com/2dust/v2rayNG/issues/2, but I don't feel sure about it. Can someone tell me what's the experience with v2rayNG and Nvidia Shield?

**Comment by w1r0x** [comment, marked as answer]
Just tested 1.8.24. Works well.

**Comment by VahidRafieiPour** [comment]
> I intend to buy an Nvidia shield but currently, I am living in China and must bypass the wall to make it work. I searched it from the threads and found a related discussion https://github.com/2dust/v2rayNG/issues/2, but I don't feel sure about it. Can someone tell me what's the experience with v2rayNG and Nvidia Shield?

**Reply by VahidRafieiPour** [reply]
Yes

**Reply by VahidRafieiPour** [reply]
بله خیلی ممنون میشم از شرکت شما که این کد از دست رفته خریداری شده ۱۰۰ گیگابایت ویتوری را که به قیمت ۴۰۰ هزار تومان به پول ایران خریداری کردم و آقای فروشنده این کد را در اختیار من قرار نمی‌دهند و ما راهی پیدا کنیم که بتونیم اثبات کنیم که این مبلغ را پرداخت کردیم و دوباره کانکشن ویتوره خودمون رو توسط لطف و عنایت شرکت شما فعال کنیم با سپاس فراوان

**Reply by VahidRafieiPour** [reply]
![Screenshot_۲۰۲۴۰۷۰۸-۰۳۵۳۱۸ (1).png](https://github.com/2dust/v2rayNG/assets/110551179/1ceb8ae5-08cd-4779-a1a8-856d195041b5)

![Screenshot_۲۰۲۴۰۷۰۸-۰۳۵۳۱۸.png](https://github.com/2dust/v2rayNG/assets/110551179/22e59c3a-56fc-4ac2-9edd-a4575fbb68d3)

من توانستم این اطلاعات را بازیابی کنم حالا به هر تقدیر به هر روشی هم که شده بتونم از ۱۰۰ گیگابایت فیلترشکن ویتوره شرکت شما استفاده کنم خیلی لطف می‌فرمایید من رو راهنمایی کنید که چطور کانکشن خود را فعال و از خدمات و سرویس شما نهایت بهره وری رو انجام دهم

**Comment by RendeRR** [comment]
Pls add it to google play
<img width="800" alt="image" src="https://github.com/2dust/v2rayNG/assets/790707/e6385601-ec50-4977-8c6b-b3bb2ba511f1">
